ai/core/pyproject.toml

```diff
@@ -11,7 +11,9 @@ license = { text="Apache-2.0" }
 requires-python = ">=3.9"
 dependencies = [
   "typing_extensions",
-  "pydantic"
+  "pydantic",
```

Comment:
Maybe add a TODO to add unitycatalog-client after it's published? Maybe we should use oss=[...] so that databricks version doesn't depend on it

Reply:
Ah great catch :D 
Adding the oss option